index.html

```diff
@@ -34,6 +34,35 @@
         <a href="#about"><p>About</p></a>
         <a href="#contact"><p>Contact</p></a>
       </nav>
+    </div>
+    <div id="overlay">s
+      <div id="popup">
+        <div class="popup-header">
+          <h4 id="popup-project-heading" class="popup-heading">This Heading</h4>
+          <i id="close-btn" href="#" onclick="PopupWindowOff()" class="fa-solid fa-close fa-2x close-popup"></i>
+        </div>
+        <div class="popup-body">
+          <div class="popup-image-area">
+            <img id="popup-project-image" alt="" />
+          </div>
+          <div class="popup-detail-area">
+            <p id="popup-project-desc"></p>
+          </div>
```

Comment:
- [ ] The width of the text should be 684px as per design. 

Your design  | Figma design
------------- | -------------
<img width="860" alt="Screenshot 2022-09-09 at 15 51 04" src="https://user-images.githubusercontent.com/77038610/189353884-33363b32-3824-4e6b-9707-917c6d738952.png">  | <img width="548" alt="Screenshot 2022-09-09 at 15 51 30" src="https://user-images.githubusercontent.com/77038610/189353941-0b85b7f9-417d-4e46-8937-3ed7efb92081.png">

Reply:
text width has been set as 684px.